draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -310,26 +303,102 @@ authorized(AS x, AS y) =  / Else, "Provider+" if the VAP-SPAS entry
         If an attacker creates a route leak intentionally, they may try to strip their AS from the AS_PATH.
         To partly guard against that, a check is necessary to match the most recently added AS in the AS_PATH to the BGP neighbor's ASN.
         This check MUST be performed as specified in Section 6.3 of <xref target="RFC4271"/>.
-        If the check fails, then the AS_PATH is considered a Malformed AS_PATH and the UPDATE is considered to be in error (Section 6.3 of <xref target="RFC4271"/>).
+        If the check fails, then the AS_PATH is considered a Malformed AS_PATH and the UPDATE is in error (Section 6.3 of <xref target="RFC4271"/>).
         The case of transparent RS MUST also be appropriately taken care of (e.g., by suspending the neighbor ASN check).
-        The check fails also when the AS_PATH is empty (zero length) and such UPDATEs will also be considered to be in error.
+        The AS_PATH check fails also when it is empty (zero length) since such an UPDATE in eBGP is considered to be an error.
```

Comment:
Looks ambigious, check is empty? 
And we are discussing only specified scenarios for inter-AS connectivity, there is no reason to add eBGP in this abstract.

Reply:
@mitradir 
Good catch. Fixed it as follows:
"If the AS_PATH is empty (zero length), then also the
   UPDATE is considered to be an error."